### PR TITLE
Feature/pb 9 categories - Rendering Categories dinamically

### PR DESCRIPTION
**Pull request by GabrielCFurlan**

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -2,5 +2,6 @@
   "semi": false,
   "trailingComma": "es5",
   "printWidth": 80,
-  "tabWidth": 2
+  "tabWidth": 2,
```
Comment (Jejugo) [inline .prettierrc]: Remove this shit

File: src/components/Categories/index.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+import computer from "../../images/Computer.png"
+import fashionMan from "../../images/Fashion_Man.png"
+import kombi from "../../images/Kombi.png"
+import pasta from "../../images/Pasta.png"
+import sunGlasses from "../../images/SunGlasses.png"
+import world from "../../images/World.png"
+
+export const categories = [
+  {
+    id: "Fashion",
```
Comment (Jejugo) [inline src/components/Categories/index.js]: use `categoryName` or even just `name` to refer to the category's name. Ids are usually used for unique identifiers.

example:
```
{
id: 1,
name: "Fashion"
}
```

File: .gitattributes [hunk 3]
```diff
@@ -0,0 +1 @@
+* text=auto eol=lf
```
Comment (Jejugo) [inline .gitattributes]: remove this shit

File: .prettierrc (hunk 1 above)
Comment (GabrielCFurlan) [reply .prettierrc]: Removed

File: .gitattributes (hunk 3 above)
Comment (GabrielCFurlan) [reply .gitattributes]: Removed

File: src/components/Categories/index.js (hunk 2 above)
Comment (GabrielCFurlan) [reply src/components/Categories/index.js]: Adjusted

**Review by Jejugo — APPROVED** [review]

**Pull request merged**